plugins/ibm_qradar/icon_ibm_qradar/util/constants/constant.py

```diff
@@ -0,0 +1,9 @@
+"""Character constants."""
+AQL = "AQL"
+SEARCH_ID = "search_id"
+POLL_INTERVAL = "poll_interval"
+HTTPS = "https"
+SUCCESS_RESPONSE_CODE = [200, 201, 206]
+
+POST = "POST"
+GET = "GET"
```

Comment:
No need for these - their usage isn't needed since the `requests` module has `post` and `get` functions.